app/controllers/reports_controller.rb

```diff
@@ -2,8 +2,21 @@
 class ReportsController < ApplicationController
   def create
     @info_request = InfoRequest.find_by_url_title!(params[:request_id])
+    if params[:comment_id]
```

Comment:
The `comment_id` param isn't persisted across a log in:

- log out
- go to request page
- click report comment
- go through log in
- Send report
- Report does not contain link to comment